packages/math-input/src/components/keypad/keypad-pages/keypad-pages.stories.tsx

```diff
@@ -1,5 +1,4 @@
 import {action} from "@storybook/addon-actions";
-import {withKnobs} from "@storybook/addon-knobs";
```

Comment:
`@storybook/addon-knobs` is deprecated.  These stories don't look like they were working properly anyways as I couldn't see any controls available in the existing, published stories: https://khan.github.io/perseus/?path=/story/math-input-components-keypad-pages--numeric-input